src/views/CreateResume.vue

```diff
@@ -20,22 +20,34 @@
       v-model="interests"
     />
 
-    <ContactInfo @contact-data="handleContactData" />
+    <ContentWrapper title="Contacts">
+      <CustomInput 
```

Comment:
Everything is great!! 

One thing I noticed, as an idea for further improvements.

<img width="183" alt="Screenshot 2024-01-26 at 12 16 05" src="https://github.com/Torishnia/resume-manager-ui/assets/48657633/a864ac0a-3c53-4364-bd72-1d912821b058">

It'd add to efficiency if there were validation rules for those inputs